extensions/cms/src/errors.rs

```diff
@@ -6,6 +6,9 @@ pub enum Error {
     /// Configuration error.
     #[error("config: {0}")]
     Config(String),
+    /// Export provider error.
+    #[error("export provider: {0}")]
+    Export(String),
```

Comment:
There are no uses of this Error on your branch.  Do we need it?

Reply:
Good catch.  Removed.